site/docs/v1/tech/apis/_application-request-body.adoc

```diff
@@ -361,6 +361,12 @@ Determines if the `mobilePhone` field will be included on the registration form.
 [field]#application.registrationConfiguration.mobilePhone.required# [type]#[Boolean]# [optional]#Optional# [default]#Defaults to `false`# [since]#Available since 1.4.0#::
 Determines if the `mobilePhone` field is required when displayed on the registration form.
 
+[field]#application.registrationConfiguration.preferredLanguages.enabled# [type]#[Boolean]# [optional]#Optional# [default]#Defaults to `false`# [since]#Available since 1.44.0#::
+Determines if the `preferredLanguages` field will be included on the registration form. The field will display all available link:/docs/v1/tech/reference/data-types#locales[locales] and is not configurable.
```

Comment:
None of the basic form values are really configurable? 

> and is not configurable.

Does this just mean you don't get to choose the values displayed? If so, this is not really accurate. You could build your own locale selector. Our default control uses `fusionAuth.locales()` in a `[#list` to iterate through all of the values, but you don't have to use this.